client/scss/components/_skiplink.scss

```diff
@@ -10,7 +10,9 @@
   }
 
   &.button {
-    background: theme('colors.positive.100');
+    background: theme('colors.secondary.DEFAULT');
     border: theme('colors.positive.100');
+   
+    
```

Comment:
please remove this extra whitespace, this is not needed here.

Reply:
Alright. I will resolve that